### ISSUE TITLE
Django is out of date

### ISSUE THREAD

**Issue by shaddi** [co-cell/ccm#1]
We're on Django 1.10, the latest Django version is 2.1.3 (support through end of Dec 2019), and the LTS release is 1.11 (support through April 2020). The upgrade from 1.10 -> 2.1 is more significant. [Details here](https://www.djangoproject.com/download/). 

My thought is to work on upgrading to the 2.1 release since that gets us basically same duration of support and prepares the project for the next LTS release, 2.2, which will have support through 2023+.  

Thoughts @matt9j @kkroo?

**Comment by eprueves** [comment]
Agree!

**Comment by matt9j** [comment]
I think we should move to 2.* as well!

**Comment by eprueves** [comment]
I think though of copying our branch (https://github.com/pcarivbts/CommunityCellularManager/tree/pcarivbts-setup/) with the role-based access and other features that we tracked from Aricent's fork  and start the Django work from there.

**Comment by matt9j** [comment]
@eprueves Is that branch the one that's in prod for you all? I think that would make sense as a baseline, since you all are the main public users of the project!

**Comment by shaddi** [comment]
Yeah, if that's what you're running in prod let's merge it in here before the update, from a quick look the 2.x switch will change a fair bit for us. Probably makes sense to get that and *maybe* the split nitb branch merged in before making more substantial changes?

**Comment by eprueves** [comment]
I've lost track what branch/version they are currently running in production, but our branch has the phase1 stuff that is definitely in prod.

**Comment by matt9j** [comment]
2.2LTS just landed, so I think this is the one to target https://www.djangoproject.com/weblog/2019/apr/01/django-22-released/
